pkg/builtinchecks/yamls/dnsconfig-options.yaml

```
@@ -0,0 +1,12 @@
+name: "dnsconfig-options"
+description: "Alert on deployments that have no specified dnsconfig options"
```

Should `dnsConfig` be camelcased to match how it appears in the spec YAML?

```suggestion
description: "Alert on deployments that have no specified dnsConfig options"
```